lib/config/presets/github.spec.ts

```diff
@@ -18,6 +18,54 @@ describe('config/presets/github', () => {
     got.mockReset();
     return global.renovateCache.rmAll();
   });
+  describe('setInternalPreset()', () => {
+    it('allows override', () => {
+      github.setInternalPreset({ body: {} });
+    });
+  });
+  describe('fetchJSONFile()', () => {
+    beforeEach(() => {
+      delete global.repoCache.internalPresets;
+    });
+    it('returns JSON', async () => {
+      hostRules.find.mockReturnValueOnce({ token: 'abc' });
+      got.mockImplementationOnce(() => ({
```

Comment:
We should use nock instead of mocking got. So we can esier identify wrong request mocking.

We should ship this fix and refactor test later. I'll add an issue to track this